src/Raven.Server/Config/Categories/BackupConfiguration.cs

```diff
@@ -64,10 +65,10 @@ public class BackupConfiguration : ConfigurationCategory
         [ConfigurationEntry("Backup.Azure.Legacy", ConfigurationEntryScope.ServerWideOnly)]
         public bool AzureLegacy { get; set; }
 
-        [Description("Compression algorithm that is used to perform backups (does not apply to snapshot backups).")]
-        [DefaultValue(ExportCompressionAlgorithm.Zstd)]
+        [Description("Compression algorithm that is used to perform backups.")]
+        [DefaultValue(BackupCompressionAlgorithm.Zstd)]
         [ConfigurationEntry("Backup.Compression.Algorithm", ConfigurationEntryScope.ServerWideOrPerDatabase)]
-        public ExportCompressionAlgorithm CompressionAlgorithm { get; set; }
+        public BackupCompressionAlgorithm CompressionAlgorithm { get; set; }
```

Comment:
this config is for 'Smuggler' backups, name enum 'BackupCompressionAlgorithm'
introduce a new option for Snapshot backups, 'Backup.Snapshot.Compression.Algorithm' with values:
- Zstd
- Deflate

Reply:
done